backend/LexBoxApi/GraphQL/UserMutations.cs

```diff
@@ -96,15 +97,16 @@ IEmailService emailService
     [Error<DbError>]
     [Error<UniqueValueException>]
     [Error<RequiredException>]
-    [AdminRequired]
     public async Task<LexAuthUser> CreateGuestUserByAdmin(
```

Comment:
let's change this name to reflect that it's no longer an admin api, same with the input class

Reply:
It's restricted to either site admins or org admins, I think the name is fine. If we changed it we'd need to change it in the frontend code as well. We could do that, but it's quite a lot of lines of code touched for a rename I'm not sure is necessary. If you still think the rename is needed, I'll do it tomorrow, but my feeling right now is that it's fine as-is.